src/java.base/share/classes/java/lang/Package.java

```diff
@@ -221,6 +221,10 @@ public String getImplementationVendor() {
     /**
      * Returns true if this package is sealed.
      *
+     * @apiNote
+     * This method reports on a distinct concept of sealing from
+     * {@link Class#isSealed() Class::isSealed}.
+     *
```

Comment:
This API note will be very confusing to readers. I think the javadoc will need to be fleshed out and probably will need to link to a section the Package class description that defines the legacy concept of sealing.

Reply:
I agree.  This @apiNote needs more clarification to help the readers to understand the context here.    One thing we could do in the Package class description to add a "Package Sealing" section that can also explain that it has no relationship to "sealed classes".

Reply:
I added an API note in `Package::isSealed` [1] to clarify sealed package vs sealed class or interface.

The API note you added in `Class::isSealed` can be clarified in a similar fashion like: "Sealed class or interface has no relationship with {@linkplain Package#isSealed package sealing}".

[1] https://github.com/openjdk/jdk/commit/3c230b8a

Reply:
Thanks for that update, Mandy! I've tweaked the API note as per your recommendation. I'll publish a fixed PR later, reflecting the other review comments as well.